client/src/components/DropdownPace/DropdownPace.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+const Select = styled.select`
+  height: 40px;
+  width: 100%;
+  margin-top: 15px;
+  border: solid 2px var(--secondary-color);
+  background: none;
+  color: var(--secondary-color);
+  font-family: "Heiti SC", adobe-heiti-std, sans-serif;
+  font-size: 1rem;
+  font-weight: 700;
+  display: flex;
+  text-align: center;
+`;
+
+const DropdownPace = ({ pace, setPace }) => {
```

Comment:
In a controlled component, better rename `setPace` to `onPaceChange`.
```suggestion
const DropdownPace = ({ pace, onPaceChange }) => {
```

https://reactjs.org/docs/forms.html#controlled-components

In addition, you could think about renaming it to `SelectPace` to have a better understanding about the rendered HTML.